src/plugins/nopSpammer.ts

```diff
@@ -0,0 +1,23 @@
+import { Devs } from "../utils/constants";
+import definePlugin from "../utils/types";
+
+export default definePlugin({
+    name: "Nop Spammer",
+    description: "makes the hidden from likely spammer tag explode",
+    authors: [
+        Devs.botato,
+        {
+            name: "Iryis",
+            id: 118437263754395652n,
+        },
+    ],
+    patches: [
+        {
+            find: "),{hasFlag:",
+            replacement: {
+                match: /(if\(e<=1<<30\)return)/,
```

Comment:
```suggestion
                match: /(if\(.{1,2}<=1<<30/,
```
then use that group in replace `if($1===`